style.css

```diff
@@ -89,3 +89,192 @@ body {
   color: white;
 }
 
+#titleimg {
+  background: url("./assets/img/projectimg.png");
+  height: 160px;
+  background-size: cover;
+  background-repeat: no-repeat;
+  margin-top: 32px;
+  margin-bottom: 42px;
+}
+
+#projecttitle {
+  margin-left: 18px;
+}
+
+.projectimg img {
+  height: 195px;
+  width: 100%;
+}
+
+.project {
+  background-color: #1a2236;
+  color: white;
+  font-family: Poppins, sans-serif;
+}
+
+.projectdetails {
+  margin-left: 16px;
+  padding-bottom: 20px;
+}
+
+.projectdetails h3 {
+  font-size: 20px;
+  font-weight: 300;
+}
+
+.projectbuttons {
+  display: flex;
+  justify-content: start;
+  gap: 10px;
+  margin-bottom: 26px;
+}
+
+.projectbuttons button {
+  background-color: #98dede;
+  color: black;
+  border: none;
+  font-size: 16px;
+  text-decoration: none;
+  text-align: center;
+  padding: 6px;
+}
+
+.viewbutton {
+  display: flex;
+  justify-content: space-around;
+  width: 55%;
+  color: white;
+  border: 2px solid white;
+  align-items: baseline;
+}
+
+.viewbutton i {
+  width: 20.57px;
+  height: 15.35px;
+}
+
+.grid-item {
+  background-color: #1c528a;
+  padding-bottom: 53px;
+  padding-left: 50px;
+  position: 50px;
+  font-size: 30px;
+  color: white;
+  font-family: Poppins, sans-serif;
+  height: 286px;
+}
+
+.grid-item h2 {
+  font-size: 20px;
+  line-height: 30px;
+  font-weight: 800;
+  padding-top: 25px;
+}
+
+.grid-item li {
+  font-size: 18px;
+  font-weight: 600;
+  line-height: 27px;
+}
+
+.gridproject {
+  display: grid;
+  grid-template-rows: repeat(2, 1fr);
+  background-color: #1a2236;
+}
+
+.gridimg {
+  grid-row: span 1;
+}
+
+.gridcontent {
+  grid-row: span 1;
+  margin-left: 16px;
+  color: white;
+  font-family: Poppins, sans-serif;
+}
+
+.gridconten h3 {
+  font-size: 20px;
+  font-weight: 300;
+}
+
+#about {
+  margin-left: 16px;
+  margin-right: 16px;
+}
+
```

Comment:
- [ ] Just like the first reviewer stated, kindly align the about with flexbox. Using flexbox helps you to align items properly and it makes your design responsive.

https://github.com/fitz95/Portfolio-Microverse-/pull/3#discussion_r1038762972

Kindly implement this change. 👍